docs/release/trg-0/trg-1-05.md

```diff
@@ -0,0 +1,35 @@
+---
+title: TRG 1.05 - Documentation Architecture
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft - MISSING EXTERNAL INFORMATION  | 10-Jan-2024  | Transfered from Release Acceptance Criteria into TRG|
+
+## Why
+
+It makes it much easier for others to use the component.
```

Comment:
line 16: We follow the good practice to document the Architecture of FOSS components. ARC42 (https://arc42.org/) is our template of choice.